### ISSUE TITLE
Disable work types for colonists

### ISSUE THREAD

**Issue by sdfg2** [LordKuper/rimworld-workmanager#4]
Could you add a section so we can disable work types for colonists?  Ta.

**Comment by LordKuper** [comment]
You can disable work type for all colonists by clicking the corresponding light bulb on the work tab, or for a single colonist by middle mouse button click on the work priority box for that particular colonist and work type.

**Comment by sdfg2** [comment]
No, I mean I want work manager to manage them, and to set them to zero.  Doing what you suggest is what I've been doing, but it then requires manually going through and setting all of them to zero.
